qa-pipelines/tasks/lib/azure-aks.sh

```diff
@@ -81,23 +82,43 @@ azure_set_a_record() {
         --ipv4-address $2
 }
 
+azure_set_cname_record() {
+    az network dns record-set cname create \
+        --resource-group ${AZURE_DNS_RESOURCE_GROUP} \
+        --zone-name ${AZURE_DNS_ZONE_NAME} \
+        --name $1 \
+        --ttl 300
+    az network dns record-set cname set-record \
+        --resource-group ${AZURE_DNS_RESOURCE_GROUP} \
+        --zone-name ${AZURE_DNS_ZONE_NAME} \
+        --record-set-name $1 \
+        --cname $2
+}
+
 azure_set_record_sets_for_namespace() {
     local namespace=$1
+    local cap_platform=$2
     local lb_info lb_ip
-    lb_info=$(kubectl get svc -n $namespace -o json | jq -c '[.items[] | select(.spec.type == "LoadBalancer") | {"svc": .metadata.name, "ip": .status.loadBalancer.ingress[0].ip}]')
+    if [[ ${cap_platform} == "eks" ]]; then
+        lb_info=$(kubectl get svc -n $namespace -o json | jq -c '[.items[] | select(.spec.type == "LoadBalancer") | {"svc": .metadata.name, "ip": .status.loadBalancer.ingress[0].hostname}]')
```

Comment:
We should use one `lb_info` definition:

```
lb_info=$(
  kubectl get svc -n $namespace -o json | 
  jq -c '
    [
      .items[] | select(.spec.type == "LoadBalancer") | {
        "svc": .metadata.name,
        "ip": .status.loadBalancer.ingress[0].ip,
        "hostname": .status.loadBalancer.ingress[0].hostname
       }
    ]'
)
```

This way you can grab `.hostname` from the `lb_info` element when using EKS, and `.ip` otherwise. For EKS, `.ip` will be null, and for non-EK, `.hostname` will be null